### PR TITLE
Increase responsiveness of Other Resources screen

### PR DESCRIPTION
**Pull request by GRardB**
I think I've come up with something good enough for MVP, but a lot of the images still need improving. I've [created a card](https://trello.com/c/qaJmt5hK/70-use-better-images-for-other-resources) to track that work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,17 +13,17 @@ import { Videos } from "./screens/videos";
 export function App() {
   return (
     <>
-      <Header />
+      {/* <Header />
```
Comment (stephan281094) [inline src/App.js]: 👀

File: src/screens/resources/resource.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          {title}
+        </h3>
+        <span>{subtext}</span>
+
+        <p className="line-clamp-3 lg:line-clamp-none">{children}</p>
+      </div>
+      <div className="flex flex-col gap-4 px-4 lg:w-56 lg:py-6 lg:pr-6">
+        {links.map(([text, url], i) => (
+          <a
+            key={i}
```
Comment (stephan281094) [inline src/screens/resources/resource.js]: Index as a key is an anti-pattern and should only be used as a last resort (explained [here](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/)). Using the `url` would be better here:
```suggestion
        {links.map(([text, url]) => (
          <a
            key={url}
```

**Review by stephan281094 — CHANGES_REQUESTED** [review]
Looking good! I only have a few small comments

File: src/screens/resources/resource.js (hunk 2 above)
Comment (GRardB) [reply src/screens/resources/resource.js]: I usually go with the array index if it's a static list, as the article says is okay:

> To help you decide, I put together three conditions which these examples have in common:
> 
> 1. the list and items are static–they are not computed and do not change;
> 2. the items in the list have no ids;
> 3. the list is never reordered or filtered.
>  
> When all of them are met, you may safely use the index as a key.

But I've changed it anyway!

**Review by stephan281094 — APPROVED** [review]

File: src/screens/resources/resource.js (hunk 2 above)
Comment (stephan281094) [reply src/screens/resources/resource.js]: The [React docs](https://reactjs.org/docs/lists-and-keys.html#keys) say:
>When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort

That's why, if something unique is available, I always use that. I'm being very nitpicky here, though!

**Pull request merged**